yt/docs/README.md

```diff
@@ -1,208 +1,434 @@
+
+2. Prepare content. To keep YTsaurus documentation consistent follow the guidelines below:
+    - [YFM Syntax Quick Reference](#yfm-syntax-quick-reference)
+    - [Style Guide](#style-guide)  
+    - [Documentation Philosophy](#documentation-philosophy)
+
+3. Add your article to the proper section in the documentation structure. Depending on the language you are writing in, you should add content to either the `ru` or `en` folder, accordingly.
+
+    Keep in mind that when adding a new article, you should follow the single source approach used in YTsaurus documentation. How to do that, shows the following example.
+  
+    Let's assume you want to add an article to the `user-guide` section to the English documentation, and let's assume you named the article file as `new-article.md`. So, to follow the single source approach you would need to do the following:
+    
+    1. Create identically named files `new-article.md` in two separate folders:
+   
+        ```
+        |-- ytsaurus/yt/docs
+            |-- en/
+                |-- toc.yaml
+                |-- user-guide/ 
+        +++         |-- new-article.md    # This file will contain a link to the content that will be included from the "_includes" folder. 
+                |-- _includes/
+                   |-- user-guide/ 
+        +++            |-- new-article.md # This file will contain the text of the article itself.
+         ```
+   
+    2. In the `en/user-guide/new-article.md` file, insert an [include](#reusing-content) construction:
+
+         ```
+         {% include [About Deployment in a Kubernetes cluster](../_includes/new-article.md) %}
+         ```
+         Note: The description set in the `[]` square brackets serves as a meta information. It does not affect the build and will not be shown on the documentation website.
+
+     3. In the `en/_includes/user-guide/new-article.md` file, write the text of your article. For example:
+
+         ```
+         # Deployment in a Kubernetes Cluster
+
+         This section describes the installation of YTsaurus in a Kubernetes cluster ...
+         ```
+        If your text contains links to another articles, see how to insert these links correctly in the [Links](#links) section.
+
+     > **Why this structure?** <br/> It is related to the specifics of the YTsaurus documentation build. Documentation is represented in two versions: public version (the one you are reading), and the internal one which is available only for the team members. Content of the public documentation can be reused in its internal version. Therefore, articles are basically placed in the `_includes` folder.
```

Comment:
Basically is probably not what you meant? I think you wanted to say something like "by default"/"as a rule" etc.